src/core/function_public.tf

```diff
@@ -0,0 +1,247 @@
+#
+# APP CONFIGURATION
+#
+
+locals {
+  function_public = {
+    app_settings_common = {
+      FUNCTIONS_WORKER_RUNTIME       = "node"
+      WEBSITE_NODE_DEFAULT_VERSION   = "14.16.0"
+      WEBSITE_RUN_FROM_PACKAGE       = "1"
+      WEBSITE_VNET_ROUTE_ALL         = "1"
+      WEBSITE_DNS_SERVER             = "168.63.129.16"
+      FUNCTIONS_WORKER_PROCESS_COUNT = 4
+      NODE_ENV                       = "production"
+
+      // Keepalive fields are all optionals
+      FETCH_KEEPALIVE_ENABLED             = "true"
+      FETCH_KEEPALIVE_SOCKET_ACTIVE_TTL   = "110000"
+      FETCH_KEEPALIVE_MAX_SOCKETS         = "40"
+      FETCH_KEEPALIVE_MAX_FREE_SOCKETS    = "10"
+      FETCH_KEEPALIVE_FREE_SOCKET_TIMEOUT = "30000"
+      FETCH_KEEPALIVE_TIMEOUT             = "60000"
+
+      COSMOSDB_URI      = data.azurerm_cosmosdb_account.cosmos_api.endpoint
+      COSMOSDB_KEY      = data.azurerm_cosmosdb_account.cosmos_api.primary_master_key
+      COSMOSDB_NAME     = "db"
+      StorageConnection = data.azurerm_storage_account.api.primary_connection_string
+
+      VALIDATION_CALLBACK_URL = "https://api-app.io.pagopa.it/email_verification.html"
+
+      SLOT_TASK_HUBNAME = "ProductionTaskHub"
```

Comment:
```suggestion
```
managed by module

Reply:
solved in https://github.com/pagopa/io-infra/pull/316/commits/e4609ce320c364770dc54708c07bbf87d24b9346